.env.sample

```
@@ -5,4 +5,13 @@ DB_HOST = LOCAL HOST
 DB_PORT = YOUT PORT
 DB_USERNAME = USERNAME
 DB_PASSWORD = PASSWORD
-DB_NAME = DATABASE
\ No newline at end of file
+DB_NAME = DATABASE
+
+KAKAO_ID = KAKAO_ID
+KAKAO_URL = REDIRECT
+
+PORT = PORT
+
+ALGORITHM=LONG
+JWT_SECRET=HIGH
```

해당 값을 유추할 수 있을 정도로 작성해주면 좋을 것 같아요!

```suggestion
ALGORITHM=ALGORITHM_HASH
JWT_SECRET=LONG_SECRET_KEY
```

---

반영 완료하였습니다.